servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Completable.java

```diff
@@ -468,7 +468,7 @@ public final Completable whenCancel(Runnable onCancel) {
      * @see <a href="http://reactivex.io/documentation/operators/timeout.html">ReactiveX timeout operator.</a>
      */
     public final Completable timeout(long duration, TimeUnit unit) {
-        return timeout(duration, unit, immediate());
+        return timeout(duration, unit, global());
```

Comment:
Ufff... an overload that does not take an explicit `Executor` makes everything harder for users who want to disable offloading. `immediate()` is also bad in this case. In future, we should consider `IoExecutor implements io.servicetalk.concurrent.Executor`

Reply:
`NettyIoExecutor` has `Executor asExecutor()` method BTW.